src/main/java/de/bonndan/nivio/input/dto/ComponentDescription.java

```diff
@@ -0,0 +1,26 @@
+package de.bonndan.nivio.input.dto;
+
+import de.bonndan.nivio.model.Labeled;
+import de.bonndan.nivio.model.Linked;
+
+/**
+ * Base interface for input DTOs.
+ *
+ *
+ */
+public interface ComponentDescription extends Labeled, Linked {
+
+    String getIdentifier();
+
+    String getName();
+
+    void setName(String name);
+
+    String getDescription();
+
+    void setDescription(String description);
+
+    String getOwner();
+
+    void setOwner(String owner);
```

Comment:
Is there any `ComponentDescription` that will not have a name, description and owner when the object is constructed? We are working towards more inmutability in objects, I would not invite to use the setters through the interface and rather deprecate as much setters as possible ;)

Reply:
Yes, it is very likely that these values become available later during processing only. I would leave the input DTOs (all what extends from ComponentDescription) as mutable as possible.

Reply:
Okay